models/users/scratch/snowplow_unified_users_this_run.sql

```diff
@@ -32,14 +32,33 @@ select
   , c.last_os_type
   , c.last_os_version
 
-  {% if var('snowplow__enable_mobile') %}
+  {% if var('snowplow__enable_mobile_context') %}
```

Comment:
I think you could keep this as two so that we are using the macro still, but then just specifying the `lasts` specifically?

Reply:
I will have to split them out of the macro as the naming convention is to rename to FIRST_ if LAST_ exists.